src/preamble.js

```diff
@@ -1177,4 +1177,7 @@ Module['asm'] = function(global, env, providedBuffer) {
 };
 #endif
 
+// Globals used by JS i64 conversions
+var tempDouble, tempI64;
```

Comment:
I find one variable per line a lot more readable.   I don't know if e follow in JS style guides but the google one seems to agree: https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration

Reply:
Sounds good, splitting to one variable per line as suggested.